### PR TITLE
News on CoRDI attendance: Create index.md

### PR DESCRIPTION
**Pull request by sebastian-schaaf**
Freiburg Galaxy Team visiting NFDI CoRDI

### CODE REVIEW

File: content/news/2023-09-12-cordi/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+[NFDI4BIOIMAGE](https://nfdi4bioimage.de) in contrast has been kicked off in March 2023, and aims for improved handling of biological imaging data. Being traditionally centric to datatypes, assays or imaging platforms, the improvement of interoperability is on the consortium's agenda. Galaxy again provides the respective infrastructure for image data handling, improving e.g. the toolset of the [imaging.usegalaxy.eu](https://imaging.usegalaxy.eu) subdomain.
+
+**At CoRDI, usegalaxy.eu was represented with two posters and two talks:**
+
+On Day 2, Sanjay presented the poster ["Conda, Container and Bots - How to build and maintain tool dependencies
+in workflows and training materials"](https://www.tib-op.org/ojs/index.php/CoRDI/article/view/417) ([PDF](https://drive.google.com/file/d/1sZm9J-U4Ets40g9QZ_J5kizx2d2xkKTd/view?usp=sharing)), and was supported by Björn and Sebastian. Lively discussions spinned up with researchers from various fields. The [durchdachte], largely bot-driven CI/CD system of Galaxy, building on Conda and Planemo, was much appreciated and regarded as a very professional approach, _including_ community-control, being open source and free of costs.
```
Comment (tuncK) [inline content/news/2023-09-12-cordi/index.md:28]: @sebastian-schaaf What was [durchdachte] supposed to refer to?

File: content/news/2023-09-12-cordi/index.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+On Day 3, Björn gave a talk on GTN and TIaaS (["FAIR and Scalable Education - The Galaxy Training Network (GTN) and a Training Infrastructure as a Service (TIaaS)"](https://www.tib-op.org/ojs/index.php/CoRDI/article/view/422)). Despite being close to Lunch, the audience of the education-related session was pleased to be provided with a 15 minutes presentation of GTN's principles, its grown contents, and not least the great community behind it. While comparable platforms address particular fields or are yet limited on the technical level or the available contents, GTN convinced with its maturity as a results of years of development. Proud to say that questions from the audience also addressed immediatly how to effectively contribute. GTN folks, be prepared for new colleagues and contents :).
```
Comment (tuncK) [inline content/news/2023-09-12-cordi/index.md:32]: Lunch -> lunch
as a results -> as a result
immediatly -> immediately

File: content/news/2023-09-12-cordi/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Right after Lunch, Sebastian presented an overview on ["Galaxy and RDM"](https://www.tib-op.org/ojs/index.php/CoRDI/article/view/421); after having introduced Galaxy as a field-agnostic platform and RDMkit's data life cycle, the [alignment with Galaxy's features](https://galaxyproject.org/news/2022-10-19-rdm-galaxy/) were sketched, underlining that Galaxy covers the entire cycle. Moreover, Galaxy improved in connecting to well-known interoperability resources like WorkflowHub and Zenodo. Standards are obviously key here, and the adoption of standardized exchange formats like RO-Crate for both importing and exporting data _and_ metadata rised considerable interest.
```
Comment (tuncK) [inline content/news/2023-09-12-cordi/index.md:34]: Lunch -> lunch
rised -> rose

File: content/news/2023-09-12-cordi/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Other unordered highlights from Karlsruhe: around 681 participants onsite, 84 talks in up to four parallel sessions, 105 posters, a very good organization team, York Sure-Vetter ("lead") and Carole Goble (Program Chair) in an excellent mood, and Björn holding a chair together with Juliane Fluck for the "Lifescience" session. If you are ever in Karlsruhe: do not miss the Light Festival show at the Castle. For the organizers' wrap up, [click here](https://www.nfdi.de/cordi-2023-5-conference-highlights/?lang=en).
```
Comment (tuncK) [inline content/news/2023-09-12-cordi/index.md:36]: life sciences

File: content/news/2023-09-12-cordi/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+[NFDI4BIOIMAGE](https://nfdi4bioimage.de) in contrast has been kicked off in March 2023, and aims for improved handling of biological imaging data. Being traditionally centric to datatypes, assays or imaging platforms, the improvement of interoperability is on the consortium's agenda. Galaxy again provides the respective infrastructure for image data handling, improving e.g. the toolset of the [imaging.usegalaxy.eu](https://imaging.usegalaxy.eu) subdomain.
+
+**At CoRDI, usegalaxy.eu was represented with two posters and two talks:**
+
+On Day 2, Sanjay presented the poster ["Conda, Container and Bots - How to build and maintain tool dependencies
+in workflows and training materials"](https://www.tib-op.org/ojs/index.php/CoRDI/article/view/417) ([PDF](https://drive.google.com/file/d/1sZm9J-U4Ets40g9QZ_J5kizx2d2xkKTd/view?usp=sharing)), and was supported by Björn and Sebastian. Lively discussions spinned up with researchers from various fields. The sophisticated, largely bot-driven CI/CD system of Galaxy, building on Conda and Planemo, was much appreciated and regarded as a very professional approach, _including_ community-control, being open source and free of costs.
```
Comment (bgruening) [inline content/news/2023-09-12-cordi/index.md:28]: citing planemo paper?

**Comment by sebastian-schaaf** [comment]
Thanks! All suggestions have been included.

**Pull request merged**